cli/azd/internal/appdetect/java.go

```diff
@@ -21,6 +21,9 @@ type mavenWrapper struct {
 	winPath   string
 }
 
+// JavaProjectOptionMavenPath The project path of the maven single-module project
+const JavaProjectOptionMavenPath = "path"
```

Comment:
Nit: rename to `JavaProjectOptionCurrentPomDir` and `JavaProjectOptionParentPomDir` will make it easier to understand. `JavaProjectOptionMavenPath` is confusing without read the codes where used it.

Reply:
Updated.